R/new_stan_data.R

```diff
@@ -7,7 +7,7 @@
 #' @param dose_cmt Specify what dose compartment. Observation compartment in 
 #' dataset is irrelevant, handled in model.
 #' @param parameters list of population parameters, e.g. `list(CL = 5, V = 50)`
-#' @param fix optional, vector of parameters to fix. These parameters will 
+#' @param fixed optional, vector of parameters to fix. These parameters will 
 #' be sampled from the posterior with negligibly narrow distribution. This is 
 #' considered a quick-and-dirty way to fix a parameter, and will slow down HMC 
 #' sampling. It should therefore only be used for testing purposes. To 
```

Comment:
```suggestion
#' be sampled from the posterior with negligibly narrow distribution. This is 
#' considered a quick-and-dirty way to fix a parameter, and will slow down HMC 
#' sampling. It should therefore only be used for testing purposes. To 
```
Is this comment still true? Isn't this the preferred way of doing it now?

Reply:
no, removing.